doc/glossary.rst

```diff
@@ -1192,7 +1192,7 @@ Methods
     ``fit_predict``
         Used especially for :term:`unsupervised`, :term:`transductive`
         estimators, this fits the model and returns the predictions (similar to
-        :term:`predict`) on the training data. In clusterers, these predictions
+        :term:`predict`) on the training data. In clusters, these predictions
```

Comment:
The original was actually correct: a clusterer is an estimator that does clustering

```suggestion
        :term:`predict`) on the training data. In clusterers, these predictions
```